go/vt/schemadiff/errors.go

```diff
@@ -7,6 +7,45 @@ import (
 	"vitess.io/vitess/go/sqlescape"
 )
 
+// Wrapped is used to unwrap an error created by errors.Join() in Go 1.20
+type Wrapped interface {
```

Comment:
this is more a structural comment; i'd suggest either renaming or relocating these types. right now you import `schemadiff.Wrapped` which, somehow, is for errors? i would expect from that name alone it's some sort of, well, wrapped diff structure (ditto the `Unwrap*` functions).

so, i'd either advocate for `schemadiff.WrappedError` or `vterrors.Wrapped` (or a new package!)

Reply:
I agree. The location is suboptimal. I initially wanted to do this in `vterrors`, but `vterrors` has its own `Unwrap`. I'll create a new `errors` package.

Reply:
Done, refactored!